src/dbt_client/dbtFusionCommandIntegration.ts

```
@@ -0,0 +1,496 @@
+        .map((line) =>
+          this.parseJSON(
+            "RebuildManifestErrorsAndWarningsJSONParsing",
+            line,
+            false,
+          ),
+        );
+      const errors = errorsAndWarnings
+        .filter(
+          (line) =>
+            line &&
+            line.hasOwnProperty("info") &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)